lib/MooX/Options/Role.pm

```diff
@@ -278,13 +278,13 @@ sub parse_options {
     # create usage str
     my $usage_str = $options_config{usage_string} // "USAGE: $prog_name %o";
 
-    my ( $opt, $usage ) = describe_options(
-        ($usage_str), @$options,
-        [ 'usage', 'show a short help message'],
-        [ 'help|h', "show a help message" ],
-        [ 'man', "show the manual" ],
-        ,@flavour
-    );
+    unshift @$options, $options_config{usage_top}          if $options_config{usage_top};
```

Comment:
This works.  But if you have a better solution I'm happy to hear it.  What kind of patch are you thinking of?